ext/simplexml/simplexml.c

```diff
@@ -250,7 +250,7 @@ static zval *sxe_prop_dim_read(zval *object, zval *member, zend_bool elements, z
 			elements = 1;
 		} else if (!member) {
 			/* This happens when the user did: $sxe[]->foo = $value */
-			php_error_docref(NULL, E_ERROR, "Cannot create unnamed attribute");
+			zend_throw_error(zend_ce_error, "Cannot create unnamed attribute");
 			return NULL;
```

Comment:
Should return `&EG(uninitialized_zval)` here as well (and other occurrence in this function).